backend/src/main/java/se/chalmers/dat265/group1/api/material/MaterialsController.java

```diff
@@ -0,0 +1,103 @@
+package se.chalmers.dat265.group1.api.material;
+
+import org.apache.commons.lang3.NotImplementedException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.apache.commons.lang3.NotImplementedException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)